### ISSUE TITLE
Big Sur icon proposal

### ISSUE THREAD

**Issue by dapetcu21** [mapeditor/tiled#2994]
With macOS Big Sur, Apple updated their [icon guidelines](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/) and started transitioning to this style of "squircle" icons:

<img width="1708" alt="Screenshot 2021-03-10 at 14 11 37" src="https://user-images.githubusercontent.com/428060/110627354-89210c80-81aa-11eb-94cb-6e43bc86080d.png">

It would be cool if Tiled conformed to the guideline for its macOS icon. I'd like to make a pull request to change the app icon and document icons (should be a matter of changing an .ico, making an Assets.car and changing some Info.plist entries), but first, let's discuss app icon ideas. 

The easy way out is to put the existing icon in a white-ish box and call it a day, but I also tried something with a blue background (which I'm not completely happy about because it turned out a bit low-contrast). Let me know what you think or if you have a better idea to try out. I'm not a exactly a designer, so feedback from people with more XP than me would be appreciated.

![Template - Icon - App - P3-](https://user-images.githubusercontent.com/428060/110628096-63483780-81ab-11eb-9d52-5c82f0aa59bd.png)
![Template - Icon - App - P3](https://user-images.githubusercontent.com/428060/110628102-6511fb00-81ab-11eb-8a13-6611fa3a525c.png)

**Comment by digitalby** [comment]
Now that macOS 26 is out, there are new Liquid Glass requirements to icons: https://developer.apple.com/design/Human-Interface-Guidelines/app-icons

Still need help adopting one? I can contribute.

**Comment by bjorn** [comment]
@digitalby Yeah, I don't think our icon looks that bad on macOS, but it definitely does not adhere to the current guidelines. I'd really appreciate if you could help with that!

**Comment by digitalby** [comment]
PR up at #4522. Kept it as a pure compliance fix: same brand-blue T, lifted onto a Liquid Glass squircle via Icon Composer. macOS 26 reads the new layered icon, macOS ≤ 15 reads the .icns auto-generated from the same source, no version bump. Three files, zero code changes. Happy to follow up on the colour-inversion variant you mentioned in 2021 if it's wanted as a separate step.
